app/experimenter/nimbus-ui/src/components/Summary/TableBranches/index.tsx

```diff
@@ -50,18 +52,18 @@ const TableBranches = ({
   const hasOneBranchNameSet = Boolean(savedBranches);
 
   return (
-    <>
+    <Card className="mt-4" border="light">
```

Comment:
Maybe instead of doing an outer 'Branches' card with multiple 'Branch' cards within, maybe just a single flat card where the header is `Branch: <branch name>`?

Something like this (but with the fonts fixed, etc)

![Screenshot 2022-11-02 at 11-42-24 Upgradable motivating software – Summary Experimenter](https://user-images.githubusercontent.com/119884/199534784-3f1cdb1e-dea5-43db-8f51-4f245f8c947e.png)

Reply:
There should definitely be one "group" with multiple "groups" within it, but I'm unsure if it should be a card or not (what that would look like visually, I gotta look it up). What I do love about Yashika's implementation is the whitespace in between the cards -- we don't want the users' eyes to glaze over while they're looking at this page; we want them to be able to find things quickly

Reply:
yeah idea was to pack them on the main component, internally it will be in cards, but let me look if I can make outer component wrapping branches better, thank you for the feedback @jaredlockhart @eliserichards 🎉